Braintree.xcodeproj/project.pbxproj

```diff
@@ -2402,6 +2440,7 @@
 				046547B22DDE8B040054B80A /* Sources */,
 				046547B32DDE8B040054B80A /* Frameworks */,
 				046547B42DDE8B040054B80A /* Resources */,
+				0453F7AF2E3D5E7900849AD8 /* Embed Frameworks */,
```

Comment:
This diff is also okay - VisaCheckout is a static library according to `file XCFrameworks/VisaCheckoutSDK.xcframework/ios-arm64/VisaCheckoutSDK.framework/VisaCheckout` which returns `VisaCheckoutSDK: Mach-O 64-bit dynamically linked shared library arm64`. With static libraries we Embed and Sign ([source](https://stackoverflow.com/questions/57687170/do-not-embed-embed-sign-embed-without-signing-what-are-they-what-th))